R/util.R

```diff
@@ -217,15 +217,16 @@ capture_log <- function(expr, filename) {
     close(con)
   })
   handle_message <- function(e) {
-    cat(e$message, file = stdout())
+    cat(crayon::strip_style(e$message), file = stdout())
   }
   handle_error <- function(e) {
     calls <- sys.calls()
     str <- utils::limitedLabels(calls[seq_len(length(calls) - 2L)])
     lab <- format(seq_along(str))
     str <- gsub("\n", sprintf("\n%s", strrep(" ", nchar(lab)[[1]] + 2)), str)
     trace <- paste(sprintf("%s: %s\n", lab, str), collapse = "")
-    cat(sprintf("Error: %s\nTraceback:\n%s", e$message, trace),
+    cat(crayon::strip_style(sprintf("Error: %s\nTraceback:\n%s",
```

Comment:
I don't think there should be style by this point (though this is likely safe)

Reply:
Thanks, have removed that